source/dyaml/parser.d

```diff
@@ -347,7 +347,7 @@ final class Parser
             while(scanner_.checkToken(TokenID.Directive))
             {
                 const token = scanner_.getToken();
-                const value = token.value;
+                string value = token.value.idup;
```

Comment:
Is the duplication really necessary here?

Reply:
for now, I believe so. There's a lot of immutable shenanigans that I'd like to eliminate in a later PR, but for now, this didn't seem to be affecting performance in my testing.